### PR TITLE
Fix creating document bug

### PR DESCRIPTION
**Pull request by tjyung**
* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Creating a new document doesn't work because the ttl input was a string when it needed to be an int.

## Related PRs or issues
[#AB69661](https://msmobilecenter.visualstudio.com/Mobile-Center/_boards/board/t/Yoann-Team/Backlog%20Items/?workitem=69661)

### CODE REVIEW

File: appcenter-data/Data.d.ts [hunk 1]
```diff
@@ -2,7 +2,6 @@
 // Licensed under the MIT License.
 
 export const enum TimeToLive {
-    INFINITE = -1,
```
Comment (jwallra) [inline appcenter-data/Data.d.ts]: Why is this removed?

Comment (tjyung) [reply appcenter-data/Data.d.ts]: The default is now infinite so was doing some cleanup but I'll put it back because its still used somewhere.
